### DISCUSSION TITLE
What are some good settings to have when training a NN

### DISCUSSION THREAD

**Discussion by cambo2015** [BrainJS/brain.js#726] [Q&A]
MY MODEL
 summary:  I am trying to train a model with 1 text input and text 1 output, specifically, the input is a product review and the output specifies if it is "good" or "bad."  I have tried LSTM and GRU

problem: it is not training properly. I keep getting  'the' repeated over and over when I run my trained network. So what makes a good Neural network? Here are some questions I have

1.  How do we know how many hidden layers to use?
2. Is a low learning rate and error threshold better?
Thanks

**Comment by robertleeplummerjr** [comment]
What network are you using, and what type of training data are you using?
